lib/components/overlays/welcome_overlay.dart

```diff
@@ -20,8 +20,11 @@ class _WelcomeOverlayState extends ShellOverlayState<WelcomeOverlay> {
 
   @override
   FutureOr<void> requestDismiss(Map<String, dynamic> args) async {
-    animationController.reverse();
-    controller.showing = false;
+    animationController.reverse().then((value) {
+      setState(() {
+        controller.showing = false;
+      });
+    });
```

Comment:
adding an await to animationController.reverse will suffice

Reply:
Ah yeah that does the trick too.  I keep overcomplicating shit lol